### PR TITLE
Added new certifications, removed old ones

### LINKED ISSUE #11424: New certificates that include number of hours
One common request is for our certificates to feature the number of hours associated with completing them. So for the current certificates, this would be 400 hours, and for the new ones, it would be 200 hours.

@atjonathan did you end up creating these certificate templates yet? Or a way that dynamically generates them with a single view, based on their type?

The 6 new certificates will be:
- Responsive Web Design
- JavaScript Algorithms and Data Structures
- Front End Libraries
- Data Visualization (note that even though this shares a name with the old certificate, it will be a new 200 hour cert)
- APIs and Microservices
- Quality Assurance and Information Security

### PR DESCRIPTION
**Pull request by ghost**
#### Pre-Submission Checklist
- [x] Branch starts with either `fix/`, `feature/`, or `translate/`.

#### Type of Change
- [x] Breaking change (fix or feature that would change existing functionality)

#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issues: Closes #11424 Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/12794

### CODE REVIEW

File: common/models/user.json [hunk 1]
```diff
@@ -139,24 +139,34 @@
...
+      "default": false
+    },
+    "isFrontEndLibsCert": {
+      "type": "boolean",
+      "description": "Camper is front end libraries certified",
+      "default": false
+    },
+    "isJsAlgoDataStructCert": {
+      "type": "boolean",
+      "description": "Campers is javascript algorithms and data structures certified",
```
Comment (Bouncey) [inline common/models/user.json]: ~~Campers~~
Camper

File: common/models/user.json [hunk 2]
```diff
@@ -139,24 +139,34 @@
...
       "default": false
     },
     "isDataVisCert": {
       "type": "boolean",
-      "description": "Camper is data visualization certified",
+      "description": "Campers is data visualization certified",
```
Comment (Bouncey) [inline common/models/user.json]: ~~Campers~~
Camper

File: common/models/user.json [hunk 3]
```diff
@@ -139,24 +139,34 @@
...
       "default": false
     },
-    "isBackEndCert": {
+    "isApisMicroservicesCert": {
       "type": "boolean",
-      "description": "Campers is back end certified",
+      "description": "Campers is apis and microservices certified",
```
Comment (Bouncey) [inline common/models/user.json]: ~~Campers~~
Camper

File: common/models/user.json [hunk 4]
```diff
@@ -139,24 +139,34 @@
...
       "default": false
     },
-    "isFullStackCert": {
+    "isInfosecQaCert": {
       "type": "boolean",
-      "description": "Campers is full stack certified",
+      "description": "Campers is information security and quality assurance certified",
```
Comment (Bouncey) [inline common/models/user.json]: ~~Campers~~
Camper

**Review by Bouncey — DISMISSED** [review]

File: common/models/user.json (hunk 1 above)
Comment (atjonathan) [inline common/models/user.json]: Fixed

File: common/models/user.json (hunk 2 above)
Comment (atjonathan) [inline common/models/user.json]: Fixed

File: common/models/user.json (hunk 4 above)
Comment (atjonathan) [inline common/models/user.json]: Fixed

**Comment by systimotic** [comment]
Should the [intro challenge](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/feature/curriculum-updated as a part of this PR?

**Comment by atjonathan** [comment]
@QuincyLarson, @BerkeleyTrue, @systimotic, @Bouncey: This should be ready to merge! I would rather keep the commits separate, as they are technically for separate things. Thanks!

File: server/utils/constantStrings.json [hunk 5]
```diff
@@ -1,6 +1,11 @@
 {
   "gitHubUserAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/29.0.1521.3 Safari/537.36",
   "frontEndChallengeId": "561add10cb82ac38a17513be",
-  "dataVisChallengeId": "561add10cb82ac38a17513b3",
-  "backEndChallengeId": "660add10cb82ac38a17513be"
+  "backEndChallengeId": "660add10cb82ac38a17513be".
```
Comment (BerkeleyTrue) [inline server/utils/constantStrings.json]: comma not dot

**Review by BerkeleyTrue — DISMISSED** [review]

File: server/utils/constantStrings.json (hunk 5 above)
Comment (atjonathan) [inline server/utils/constantStrings.json]: Fixed

**Comment by sithsupersu** [comment]
I need to read more. I still dont know what to do!?

**Comment by QuincyLarson** [comment]
@atjonathan This is on the short list of PRs we'll merge into staging as soon as the Curriculum Expansion branch is ready, and before it's deployed. For now I'm switching this to "blocked"

**Comment by atjonathan** [comment]
@QuincyLarson, no problem. This PR was however aimed at the Curriculum Expansion branch?

**Comment by QuincyLarson** [comment]
@atjonathan I think we should leave it as is. The curriculum expansion branch is mostly the same as staging anyway - we're just touching the seed files.

**Comment by atjonathan** [comment]
@QuincyLarson 👍 🎉

**Comment by atjonathan** [comment]
@QuincyLarson, is this ready to be reopened and merged?

**Comment by QuincyLarson** [comment]
@atjonathan I tried to rebase staging onto this to fix the failing tests. Unfortunately, there were literally dozens of conflicts. The seed files have moved to an entirely different directory. 

I think it would be easier for you to branch off staging and re-implement this.

**Comment by atjonathan** [comment]
@QuincyLarson, sure, yeah this was done over 3 months ago... I'll recreate this PR along with the other things needing to be done mentioned in the issues you created 😄

**Comment by QuincyLarson** [comment]
@atjonathan awesome! Yes, thank you for your patience and help!

**Comment by sweetnlow0131** [comment]
I didn't post these questions & problems so my account is hacked now due to these changes made to my browsers!!!  So how do I undo all this to remove from all my browsers?

**Comment by QuincyLarson** [comment]
@atjonathan any update on this?

**Pull request closed without merging**